data-service/README.md

```diff
@@ -41,3 +41,76 @@ To run Prettier
 ```
 npm run pretty-quick
 ```
+
+## Build Docker container
+
+```
+docker build -t core-banking/data-service .
+```
+
+## GraphQL queries
+
+```
+query Client($id:String!) {
+  client(id:$id) {
+    clientId
+    firstName
+    lastName
+    socialSecurityNumber
+    sex
+    dateOfBirth
+    accounts {
+      accountId
+      frequency
+      createDate
+      transactions {
+        accountId
+        accountIban
+        type
+        amount
+        balance
+        amount
+        transId
+        }
+    }
+  }
+}
+
+query Accounts($id:String!) {
+  accounts(clientId:$id) {
+    accountId
+    frequency
+    createDate
+    transactions {
+        accountId
+        accountIban
+        type
+        amount
+        balance
+        amount
+        transId
+    }
+  }
+}
+
+query Transactions($accountId:Float!) {
+  transactions(accountId:$accountId) {
+    accountId
+    accountIban
+    type
+    amount
+    balance
+    amount
+    transId
+  }
+}
+```
+
+### query variables
+
+```
+{
+  "id":"B12456",
+  "accountId": 12286356
+}
+```
```

Comment:
Could you please add usage example to illustrate how to use this service?
Just a sample request and response to see how the query works. Thanks!